modules/mint-mode.adoc

```diff
@@ -5,21 +5,15 @@
 // * installing/installing_gcp/manually-creating-iam-gcp.adoc
 
 [id="mint-mode_{context}"]
-= Mint Mode
+= Mint mode
 
-Mint Mode is supported for AWS, GCP, and Azure.
+Mint mode is the default and recommended Cloud Credential Operator (CCO) credentials mode for {product-title}. In this mode, the CCO uses the provided administrator-level cloud credential to run the cluster. Mint mode is supported for AWS, GCP, and Azure.
 
-The default and recommended best practice for running {product-title} is to run
-the installer with an administrator-level cloud credential. The `admin` credential is
-stored in the `kube-system` namespace, and then used by the Cloud Credential
-Operator to process the `CredentialsRequest` objects in the cluster and create new users
-for each with specific permissions.
+In mint mode, the `admin` credential is stored in the `kube-system` namespace and then used by the CCO to process the `CredentialsRequest` objects in the cluster and create users for each with specific permissions.
 
-The benefits of Mint Mode include:
+The benefits of mint mode include:
 
-* Each cluster component only has the permissions it requires.
-* Automatic, on-going reconciliation for cloud credentials including upgrades,
-which might require additional credentials or permissions.
+* Each cluster component has only the permissions it requires
+* Automatic, on-going reconciliation for cloud credentials including upgrades, which might require additional credentials or permissions
```

Comment:
Small revision suggestion, mostly to clarify that it is "upgrades" that might require additional perms, and not "cloud credentials including upgrades" (if that's correct):

> Automatic on-going reconciliation for cloud credentials, including upgrades that might require additional credentials or permissions

Reply:
Thank you for highlighting this. On review, I have updated the sentence to state:

"Automatic, on-going reconciliation for cloud credentials, including any additional credentials or permissions that might be required for upgrades"